### PR TITLE
Docs: AV1

### PR DESCRIPTION
**Pull request by abhinavr4**

### CODE REVIEW

File: features/video-optimization/automatic-video-format-conversion.md [hunk 1]
```diff
@@ -12,7 +12,9 @@ VP9 is an open and royalty-free video coding format developed by Google. It prov
 
 ![VP9 support - https://caniuse.com/](<../../.gitbook/assets/image (29).png>)
 
-ImageKit chooses between H.264 and VP9 codec and delivers the video in the appropriate format automatically based on browser support.
+ImageKit chooses between H.264, VP9 and AV1 codec and delivers the video in the appropriate format automatically based on browser support.
+
+Note that the automatic usage of the AV1 codec for output videos is disabled by default. However, you can force the output video to use the AV1 codec by adding `vc-av1` in the transformation string. You can reach out to support@imagekit.io to get this enabled.
```
Comment (niveditn) [inline features/video-optimization/automatic-video-format-conversion.md:19]: Please add an info banner notifying the beta status of this feature

Comment (abhinavr4) [reply features/video-optimization/automatic-video-format-conversion.md:19]: Done

File: features/video-optimization/automatic-video-format-conversion.md [hunk 2]
```diff
@@ -12,10 +12,17 @@ VP9 is an open and royalty-free video coding format developed by Google. It prov
```
Comment (niveditn) [inline features/video-optimization/automatic-video-format-conversion.md:12]: > VP9 is an open and royalty-free video coding format developed by Google. It provides excellent compression and is supported by major browsers.

@abhinavr4 Should we add a similar text for AV1?

Comment (abhinavr4) [reply features/video-optimization/automatic-video-format-conversion.md:12]: Added

**Review by imagekitio — CHANGES_REQUESTED** [review]
Mention 10x vpu multiplier

**Review by niveditn — APPROVED** [review]
Made a small grammatical fix in video-optimization doc.

**Review by imagekitio — APPROVED** [review]

**Pull request merged**